integration_tests/configs/utils.jsonnet

```diff
@@ -0,0 +1,9 @@
+{
+  validator(mnemonic):: {
+    coins: '1000000000000000000stake,10000000000000000000000basetcro',
+    staked: '1000000000000000000stake',
+    mnemonic: mnemonic,
+  },
+  validators(mnemonics):: [self.validator(mnemonic) for mnemonic in mnemonics],
+  accounts(objs):: [obj for obj in objs],
```

Comment:
https://github.com/crypto-com/pystarport/pull/57#pullrequestreview-1024207540
similar opinions on this.

Reply:
removed